pages/common/s3cmd.md

```diff
@@ -0,0 +1,32 @@
+# s3cmd
+
+> Command line tool and client for uploading, retrieveing and managing data in S3 compatible object storage.
+> More information: <https://s3tools.org/s3cmd>.
+
+- Invoke configuration/reconfiguration tool:
+
+`s3cmd --configure`
+
+- List Buckets/Folders/Objects:
+
+`s3cmd ls s3://{{bucket|path/to/file}}`
+
+- Make Bucket/Folder:
```

Comment:
Do you mean `create` a Bucket here?

Reply:
Yes, "create" would be a better work. Thanks :)